### DISCUSSION TITLE
Roadrunner specific version configuration reference

### DISCUSSION THREAD

**Discussion by jelovac** [roadrunner-server/roadrunner#1025] [Q&A] [answered]
Hi,

Is there documentation where I can find configuration reference for specific RR version? Like for example what is valid YAML configuration for 2.6, 2.7, 2.8 etc.

The only thing I found regarding the config reference is this: https://roadrunner.dev/docs/intro-config . And that seems to be the config for the latest version.

Since the format of the config which RR uses is YAML and YAML doesn't have a schema it is very hard to figure out what are all the available options which can be specified if they are not properly documented.

**Comment by rustatian** [comment, marked as answer]
Hey @jelovac, you can find the configuration compatibility matrix here: https://roadrunner.dev/docs/beep-beep-config.
Also, we always notify our users about configuration changes in the release notes, like [here](https://github.com/roadrunner-server/roadrunner/releases/tag/v2.7.0) or [here](https://github.com/roadrunner-server/roadrunner/releases/tag/v2.8.0).

**Reply by rustatian** [reply]
The configuration reference with the per-key documentation you can find here: https://github.com/roadrunner-server/roadrunner/blob/master/.rr.yaml

**Reply by jelovac** [reply]
This helps, thank you for the quick reply!

**Reply by rustatian** [reply]
You are welcome 👍🏻
